.gitignore

```diff
@@ -11,3 +11,5 @@ go1.x/run/aws-lambda-mock
 go1.x/run/vendor
 examples/go1.x/handler
 examples/go1.x/vendor
+dotnetcore2.0/run/bin
+dotnetcore2.0/run/obj
```

Comment:
In addition to the two you have above.

```
dotnetcore2.0/run/bin
dotnetcore2.0/run/obj
```


Can you add the following to `.gitignore`?

```diff
node_modules
base/diff
*.pyc
.gradle
+dotnetcore2.0/run/MockBootstraps/obj
java8/run/lambda-runtime-mock/target
+base/dump-dotnet2/bin
+base/dump-dotnet2/obj
base/dump-java8/target
+examples/dotnetcore2.0/test/bin
+examples/dotnetcore2.0/test/obj
+examples/dotnetcore2.0/test/pub
examples/java/build
examples/java/target
examples/java/bin
go1.x/run/aws-lambda-mock
go1.x/run/vendor
examples/go1.x/handler
examples/go1.x/vendor
dotnetcore2.0/run/bin
dotnetcore2.0/run/obj
```